server/src/main/java/org/jboss/as/server/controller/resources/ServerSubDeploymentResourceDefinition.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 Red Hat, Inc.
```

Comment:
@yersan fiy you should update your header template to 2019 (this PR can keep them at 2018, that's ok)

Reply:
Ok, yes, you're right, thank!